R/addins.R

```diff
@@ -5,7 +5,8 @@ addin_lint <- function() {
   }
   filename <- rstudioapi::getSourceEditorContext()
   if (filename$path == "") {
-    return("Current source has no path. Please save before continue")
+    warning("Current source has no path. Please save before continuing.", call. = FALSE)
```

Comment:
shall we wait for the {cli} PR first?

Reply:
I think this issue is independent of that PR. We can just convert that warning to cli warning in the PR once this change is merged to main.